spec/support/integration_test_runner.rb

```diff
@@ -19,7 +26,9 @@ def initialize(codeowners: [], owners: [], file_tree: {}, flags: [], answers: []
   # rubocop: disable Lint/HandleExceptions
   def run
     setup_project
+    $stdout = CaptureIO.new
     setup_io_listeners
+    $stdout = STDOUT
```

Comment:
👍 nice trick! I was thinking about how to do that without affecting the whole application 👍